packages/glimmer-runtime/lib/compiled/opcodes/builder.ts

```diff
@@ -59,6 +59,14 @@ class StatementCompilationBufferProxy implements StatementCompilationBuffer {
   hasBlockSymbol(name: string): boolean {
     return this.inner.hasBlockSymbol(name);
   }
+
+  getPartialArgsSymbol(): number {
+    return this.inner.getPartialArgsSymbol();
+  }
+
+  hasPartialArgsSymbol(): boolean {
+    return this.inner.hasPartialArgsSymbol();
+  }
```

Comment:
I wanted to kill `SymbolLookup` and just use the symbol table, but ran out of time. See https://github.com/emberjs/ember.js/blob/master/packages/ember-glimmer/lib/helpers/debugger.js#L70-L74.

I believe the main reason for SymbolLookup to exist was that there use to be a "keyword" feature whose information came from the env. But now that the feature is gone, I think all this is doing is just delegating to the symbol table.